### ISSUE TITLE
Regression: !important isn't being handled properly

### ISSUE THREAD

**Issue by ngyikp** [jakubpawlowicz/clean-css#660]
```css
/**
 * multiple border-color properties are wrongly combined if !important
 * 
 * actual result:
 *   .testA{border-color:#456 #123!important #123!important}
 */
.testA {
	border-color: #123 !important;
	border-top-color: #456 !important;
}

/**
 * the first !important gets dropped even though it shouldn't
 * 
 * actual result:
 *   .testB1,.testB3{margin:0}.testB1{margin:1px!important}
 */
.testB1, .testB3 {
	margin: 0 !important;
}
.testB1 {
	margin: 1px !important;
}
```

```
Ngs-Applejack:cleancsstest ngyikp$ node --version
v0.12.7

Ngs-Applejack:cleancsstest ngyikp$ npm install clean-css@3.3.10
clean-css@3.3.10 node_modules/clean-css
├── commander@2.8.1 (graceful-readlink@1.0.1)
└── source-map@0.4.4 (amdefine@1.0.0)
Ngs-Applejack:cleancsstest ngyikp$ ./node_modules/clean-css/bin/cleancss --version
3.3.10
Ngs-Applejack:cleancsstest ngyikp$ ./node_modules/clean-css/bin/cleancss test.css
.testA{border-color:#456 #123 #123!important}.testB1,.testB3{margin:0!important}.testB1{margin:1px!important}

Ngs-Applejack:cleancsstest ngyikp$ npm install clean-css@3.4.1
clean-css@3.4.1 node_modules/clean-css
├── commander@2.8.1 (graceful-readlink@1.0.1)
└── source-map@0.4.4 (amdefine@1.0.0)
Ngs-Applejack:cleancsstest ngyikp$ ./node_modules/clean-css/bin/cleancss test.css
.testA{border-color:#456 #123!important #123!important}.testB1,.testB3{margin:0}.testB1{margin:1px!important}
```

**Comment by jakubpawlowicz** [comment]
Good catch @ngyikp :+1: - fixes are coming.

**Comment by jakubpawlowicz** [comment]
@ngyikp it's fixed in 3.4.2 now. Thanks for reporting both issues!
